extensions/dicom-pdf/src/id.js

```diff
@@ -0,0 +1,9 @@
+const Name = 'dicom-pdf';
+const id = `org.ohif.${Name}`;
+
+export default id;
+
+const SOPClassHandlerId = `${id}.sopClassHandlerModule.${Name}`;
+const ViewportModuleId = `${id}.viewportModule.${Name}`;
```

Comment:
I don't think the viewportModuleId is used anywhere either. I searched an there is none. dicom-video has this also which should be deleted too